backend/users/forms.py

```diff
@@ -58,3 +63,57 @@ def save(self):
         user = authenticate(email=email, password=password)
         login(self.context, user)
         return user
+
+
+class UpdateForm(ContextAwareModelForm):
+    date_birth = CharField(required=False)
+    # TODO: Why if I tell you required=FALSE, the mutation fail if I don't put
+    # Bool! (instead of 'Bool')?
+    open_to_recruiting = BooleanField(required=False)
```

Comment:
I think if you check the mapping it says that BooleanField is always mapped to a required parameter

Reply:
Ah oks, i removed the todo...